service/gcs/bridge/connection.go

```diff
@@ -5,48 +5,18 @@ import (
 
 	"github.com/pkg/errors"
 
+	"io"
```

Comment:
It looks like goimports will preserve your manual separation if you make one, which is probably why this one got screwed up. But in general I never manually write imports because goimports usually finds the right one for me, so that's why I like their default behavior. If you're using vscode, you can just set a configuration option to use goimports when saving instead of gofmt.